server/routes/signup.tsx

```diff
@@ -19,6 +19,7 @@ app.get('/signup', (req, res, next) => {
 						title: `Signup · ${initialData.communityData.title}`,
 						description: initialData.communityData.description,
 					})}
+					bodyClassPrefix="app"
```

Comment:
Here too I am unsure whether we need/want to provide a class that will never be targeted by custom CSS.

Reply:
I agree for www and dashboard. But I can see app pages being targeted by custom CSS in the future (e.g. you want to make footer mods across your site). We don't have to do it with this project, though.